### PR TITLE
Make XDT PC only more clear

### LINKED ISSUE #1907: XDT not supported for PackageReference projects
According to https://docs.microsoft.com/en-us/nuget/consume-packages/migrate-packages-config-to-package-reference, XDT transforms are not supported for PackageReference projects. Shouldn't this be mentioned here??

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 40bb9dee-46d6-4064-83e8-ba2e22ac6afe
* Version Independent ID: aac0bbd8-645b-ccf8-bec9-e0d970c8b089
* Content: [Source and config file transformations for NuGet packages](https://docs.microsoft.com/en-us/nuget/create-packages/source-and-config-file-transformations)
* Content Source: [docs/create-packages/Source-and-Config-File-Transformations.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/create-packages/Source-and-Config-File-Transformations.md)
* Product: **nuget**
* GitHub Login: @karann-msft
* Microsoft Alias: **karann**

### PR DESCRIPTION
**Pull request by zivkan**
fixes #1907

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[95a8020](https://github.com/NuGet/docs.microsoft.com-nuget/commits/95a8020ac732b61b61a019806a99511f00a4d5bb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Source-and-Config-File-Transformations.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/zivkan-xdt-pc-only/docs/create-packages/Source-and-Config-File-Transformations.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/source-and-config-file-transformations?branch=pr-en-us-1914) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281717384030-1914%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=0264sueM%2B9Z7uYVZEGff%2Bxnv0k5S2Zy3Wq8uw9yOmu4%3D&st=2020-02-28T17%3A13%3A18Z&se=2020-03-30T17%3A18%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zivkan** [comment]
more than I'm willing to do right now.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[95a8020](https://github.com/NuGet/docs.microsoft.com-nuget/commits/95a8020ac732b61b61a019806a99511f00a4d5bb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Source-and-Config-File-Transformations.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/zivkan-xdt-pc-only/docs/create-packages/Source-and-Config-File-Transformations.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/source-and-config-file-transformations?branch=pr-en-us-1914) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C2%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003022010228530-1914%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dcruI5s8SZsq9YYwQ5cJJTI%2FoiVhdGolMQhnUuhLFL4%3D&st=2020-03-02T20%3A06%3A04Z&se=2020-04-02T20%3A11%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Source-and-Config-File-Transformations.md [hunk 1]
```diff
@@ -109,6 +107,9 @@ To see the effect of installing and uninstalling the package, create a new ASP.N
 
 ### XDT transforms
 
+> [!Note]
+> As mentioned in the [package compatibility issues section of the docs for migrating from `packages.config` to `PackageReference`](../consume-packages/migrate-packages-config-to-package-reference.md#package-compatibility-issues), XDT transformations are only supported by `packages.config`. If you add the below files to your package, consumers using your package with `PackageReference` will not have the transformations applied.
```
Comment (karann-msft) [inline docs/create-packages/Source-and-Config-File-Transformations.md]:
```suggestion
> As mentioned in the [package compatibility issues section of the docs for migrating from `packages.config` to `PackageReference`](../consume-packages/migrate-packages-config-to-package-reference.md#package-compatibility-issues), XDT transformations as described below are only supported by `packages.config`. If you add the below files to your package, consumers using your package with `PackageReference` will not have the transformations applied (refer to [this sample](https://github.com/NuGet/Samples/tree/master/XDTransformExample) to make XDT transforms work with`PackageReference`).
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[ec1f447](https://github.com/NuGet/docs.microsoft.com-nuget/commits/ec1f44702735638acf683ff07ebc5de5f030925f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Source-and-Config-File-Transformations.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/zivkan-xdt-pc-only/docs/create-packages/Source-and-Config-File-Transformations.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/source-and-config-file-transformations?branch=pr-en-us-1914) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C2%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003022019307991-1914%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=W9Q%2B1PdOdDNAI25TdRFmz6bPTrv1DMopfIEcL83x4dc%3D&st=2020-03-02T20%3A15%3A11Z&se=2020-04-02T20%3A20%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**